### DISCUSSION TITLE
Make clicking /colorpicker ADD to the text and not wipe it!

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#6734] [Enhancement]
![2022-03-14_22 09 00](https://user-images.githubusercontent.com/144046/158270466-38f99c57-1c3f-4f23-869d-25860366f007.png)
When you click in /colorpicker it wipes your chat, so you cant use multiple colors together when clicking! This means it cant be used for items either, simply allow us to click and it adds to the end of the chat message.

**Comment by mercurialmusic** [comment]
You can do this already with a shift+click. ;)
